charts/unpoller/values.yaml

```diff
@@ -0,0 +1,118 @@
+# Default values for unpoller.
+
+replicaCount: 1
+
+image:
+  repository:  "ghcr.io/unpoller/unpoller"
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v2.11.2"
```

Comment:
might actually make more sense to just leave it as `"v2"` this should update with the semantic versioning